src/entity/CommonGame.java

```diff
@@ -0,0 +1,102 @@
+package entity;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class CommonGame implements Game {
+    private final String ID;
+    private int score;
+    private final String difficulty;
+    private final String genre;
+    private final int initialLives;
+    private final int maxRounds;
+    private int currentLives;
+    private int roundsPlayed = 0;
+    private ArrayList<Round> allRounds;
+    private final LocalDateTime createdAt;
+    private LocalDateTime finishedAt;
+
+    CommonGame(String genre, String difficulty, int maxRounds,
```

Comment:
Looking over this again, should there be public CommonGame?

Reply:
Yes.